### PR TITLE
Add typings for ent.

### PR DESCRIPTION
**Pull request by rogierschouten**

### CODE REVIEW

File: ent/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// Type definitions for ent 2.2.0
+// Project: https://github.com/substack/node-ent
+// Definitions by: Rogier Schouten <https://github.com/rogierschouten>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference types="node" />
+declare module "ent" {
```
Comment (mhegazy) [inline ent/index.d.ts]: remove `declare module "ent" {`

File: ent/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+		/**
+		 * If opts.numeric is false or opts.named is true, encoding will used named codes like &pi;.
+		 */
+		named?: boolean;
+		
+		/**
+		 * If opts.special is set to an Object, the key names will be forced to be encoded (defaults to forcing: <>'"&)
+		 */
+		special?: Object;
```
Comment (mhegazy) [inline ent/index.d.ts]: you probably mean `any`. if not then make `{ }` instead of `Object`.

**Review by mhegazy — CHANGES_REQUESTED** [review]

**Comment by rogierschouten** [comment]
@mhegazy done. Thank you for your comments!

**Comment by mhegazy** [comment]
I have a fix for the failing CI tests in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11835

**Review by mhegazy — APPROVED** [review]

**Comment by mhegazy** [comment]
thanks!

**Pull request merged**